src/instructlab/generator/generate_data.py

```diff
@@ -302,6 +337,7 @@ def generate_data(
     logger,
     api_base,
     tls_insecure,
+    model_family: str,
```

Comment:
Do we need to introduce a new `model_family`? Can we not interpret the model family from `model_name`? 
Name: `merlinite-4b-asdf.gguf`. Family: `merlinite`
Name: `granite-4b-asdf.gguf`. Family: `merlinite` 
Name: `mixtral-4b-asdf.gguf`. Family: `mixtral` 

This solves for what Mo commented too, without any additional code changes.

Reply:
Was there a resolution to this? I'm curious about this point as well

Reply:
Nah I was told ignore for this PR it'll be a follow up 🤷🏽